jerry-core/debugger/debugger.c

```diff
@@ -202,6 +218,23 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
 } /* jerry_debugger_send_eval */
 
 /**
+ * Suspend execution for received time
+ */
+static void
+jerry_debugger_sleep (int milliseconds) /**< Suspending time */
```

Comment:
usually, we don't capitalize the docstring of parameters (see rest of the code base, or this file).